fastlane/Fastfile

```diff
@@ -263,30 +283,33 @@ platform :ios do
   #####################################################################################
   desc 'Trigger the final release build on CI'
   lane :finalize_release do |options|
+    UI.user_error!('To finalize a hotfix, please use the finalize_hotfix_release lane instead') if ios_current_branch_is_hotfix
+
     ios_finalize_prechecks(options)
```

Comment:
The only change in the lines below this one is that when the `unless ios_current_branch_is_hotfix` test was removed, the rest of the code got un-indented. You might want to review those lines with "ignore whitespaces" turned on for ease of review.